react-native-community/react-native-picker
# chore: Rewrite examples

# Summary

- Merge the two examples, update the example to use hooks.
- Converted example to TypeScript.
- Add npmignore: Reduce packed files from 162 files to 65 files.
- Reduced packaged size from 714.9 kB to 28.7 kB (~25x smaller)



## Test Plan

- Run all examples (only could test iOS, Android, and web).
- Ensure pack isn't adding extra files.
- TS should work.



### What's required for testing (prerequisites)?

a computer, iOS and Android devices.

### What are the steps to reproduce (after prerequisites)?

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |
| Web |    ✅     |

## Checklist
